src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,18 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  if (user) {
+    const {
+      email,
+      name,
+    } = user;
+
+    return (
+      <a className="UserInfo" href={email ? `mailto:${email}` : '#'}>
```

Comment:
The class name 'UserInfo' does not conform to BEM naming conventions. Consider using a more descriptive class name.